templates/lx_userdata.sh

```diff
@@ -237,6 +237,14 @@ install-docker() {
   apt-get -y install docker-ce docker-ce-cli containerd.io
 }
 
+# shellcheck disable=SC2317
```

Comment:
That's a curious one. Wonder why it thinks the code is unreachable?

Reply:
Yeah.  No idea myself.

Reply:
Maybe the `try_cmd` usage?